mathesar_ui/src/component-library/text-input/TextInput.svelte

```diff
@@ -25,9 +29,22 @@
       element?.focus();
     }
   }
+
+  function handleKeypress(e: KeyboardEvent) {
+    if (e.key === 'Enter') {
+      dispatch('enter');
+    }
+  }
```

Comment:
I think it would be better to bubble up the keypress event directly rather than handling it within the TextInput component.

Reply:
We can handle this on a subsequent PR.